app/cdap/components/Connections/Browser/SidePanel/CategorizedConnections.tsx

```diff
@@ -156,6 +183,105 @@ export function CategorizedConnections({
     }
   };
 
+  function getConnectionConfig(conn) {
+    const connectionConfig = {
+      name: conn.name,
+      description: conn.description,
+      category: conn.plugin.category,
+      plugin: conn.plugin,
+    };
+
+    return connectionConfig;
+  }
+
+  function exportConnection(conn) {
+    const connectionConfig = getConnectionConfig(conn);
+    const artifact = objectQuery(conn, 'plugin', 'artifact') || {};
```

Comment:
Comment only: `conn?.plugin?.artifact || {}` will preserve types more accurately. Might be possible to get objectQuery to preserve the types, not sure how much typescript's type system allows there.